CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 - Update tblproperties on incremental runs. Note: only adds/edits. Deletes are too risky/complex for now ([765](https://github.com/databricks/dbt-databricks/pull/765))
 - Update default scope/redirect Url for OAuth U2M, so with default OAuth app user can run python models ([776](https://github.com/databricks/dbt-databricks/pull/776))
 
+### Features
```

Comment:
At this point we're holding new features for 1.9.  Can you try rebasing your PR against 1.9.latest branch, and change PR target to there?

Reply:
will do :)